src/elements/content-sidebar/SidebarContent.scss

```diff
@@ -35,12 +40,20 @@
 
     .bcs-scroll-content {
         // Styles can be overriden by each sidebar
-        width: 323px; // 17px less than the total width for scroll bar
+        width: $sidebarScrollContentWidth; // 17px less than the total width for scroll bar
+    }
+}
+
+.bcs-wider .bcs-content {
+    width: $sidebarIncreasedWidth;
+
+    .bcs-scroll-content {
+        width: $sidebarScrollContentIncreasedWidth;
     }
 }
 
 @include breakpoint($medium-screen) {
-    .bcs-content {
+    .bcs-wider .bcs-content {
```

Comment:
With this change we're replacing the original selector, which means the styling would not be applied if `.bcs-wider` class is not present. Should this be
```
.bcs-content,
.bcs-wider .bcs-content {
```
?